db/models.py

```diff
@@ -1 +1,23 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.CASCADE)
+
+class Guild(models.Model):
+    name = models.CharField(max_length=255, unique=True, null=True)
+    description = models.TextField(null=True)
+
+class Player(models.Model):
+    nickname = models.CharField(max_length=255, unique=True)
+    email = models.EmailField(max_length=255, unique=False)
```

Comment:
Consider setting `unique=True` for the `email` field in the `Player` model if you want to ensure that each email address is unique across all players.